routes/index.js

```diff
@@ -1,10 +1,29 @@
 var express = require('express');
 var router = express.Router();
+var json = require('./data.json')
+
+// need 4 elements from the json array in blog.ejs file
+let data = []
+
+for (let i=0; i<=3; i++) {
+  let release = json[i].release
+
+  let date = json[i].release.split("T")[0].replace(/-/g, '/')
```

Comment:
Instead of doing this, there's a library called `momentjs` check it out.

Reply:
Great package indeed, added in commit [04a095](https://github.com/zairza-cetb/zairza-website/pull/73/commits/04a095ad9b786f91340497477a0377385293d885) :tada: